releasenotes/notes/use-bootstrap-user-variables-template-var-080d52eb63fb0f84.yaml

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
Again, this is a test only thing. I don't believe that it belongs in release notes which are targeted at Operators.